tests/test_apertures.py

```diff
@@ -0,0 +1,70 @@
+# See COPYRIGHT file at the top of the source tree.
```

Comment:
I don't think you need this line, e.g. https://github.com/lsst/templates/blob/master/file_templates/stack_license_preamble_py/example.py